js/src/tooltip.js

```diff
@@ -590,6 +590,10 @@ class Tooltip {
 
     if (title || originalTitleType !== 'string') {
       this.element.setAttribute('data-bs-original-title', title || '')
+      if (title && !this.element.getAttribute('aria-label') && !this.element.textContent) {
+        this.element.setAttribute('aria-label', title || '')
```

Comment:
Logical OR `||` operator seems unnecessary here 🤔 
Because the `title` would never be falsy value here it is already checked in the condition above.